pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go

```diff
@@ -80,6 +80,7 @@ func verifyIngestionStats(t *testing.T, streamID int64, cutoverTime time.Time, s
 func TestTenantStreaming(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+	skip.WithIsue(t, 82706)
```

Comment:
I'm bad at code review.
```suggestion
	skip.WithIssue(t, 82706)
```